Benabout

benblack769 opened the pull request:
- Fixed text being squished to side of image for small screen sizes
- Fixed text going all the way to the edge of the screen on large screens.
- Cleaned up code some (deleted absolute attributes, which allowed for some other deletions).

benblack769 committed:
removed absolute parameters, made inline images work much nicer

benblack769 committed:
made a max text width for about page

benblack769 committed:
cleaned up css

benblack769 committed:
got rid of horizontal scrolling!

kamba16 commented:
- increase padding on text for mobile
- change name `.no-horz-scroll` to `.wrapper` (because that's what it is)

benblack769 commented:
- changed to .about-wrapper so it doesn't conflict with footer.css's .wrapper.
- think I fixed it
  ![benabout](https://cloud.githubusercontent.com/assets/10835669/17419894/78766740-5a54-11e6-9fbc-cb932a7d1e41.png)

benblack769 committed:
fixed padding and naming

benblack769 committed:
fixed about image paths

benblack769 committed:
fixed urls for about images

benblack769 commented:
Just moved the about files from static/uploads/about to static/about.

kamba16 commented:
We can probably delete most of those about images since they aren't used

El jueves, 4 de agosto de 2016, weepingwillowben notifications@github.com
escribió:

> Just moved the about files from static/uploads/about to static/about.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> https://github.com/reed-college/strabo/pull/43#issuecomment-237701337,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AQliqHzzcGh268zcoY0mp7LhJZ0Pd0H4ks5qcmYJgaJpZM4JcNcn
> .
